Orange/widgets/utils/messages.py

```diff
@@ -325,16 +325,16 @@ def update_message_state(self):
             self._hide_message_bar()
             return
         else:
+            font_size = self.message_bar.fontInfo().pixelSize()
             group = messages[0].group
             text = str(messages[0]) if len(messages) == 1 \
                 else "{} problems during execution".format(len(messages))
             # TODO: fix tooltip background color - it is not white
-            tooltip = '<div style="background-color: #ffffff;">{}</div>'.format(
-                ''.join(
-                    '<p style="background-color: {}; padding: 5">'
-                    '<nobr>{}</nobr></p>'.format(
-                        msg.group.bar_background, str(msg))
-                    for msg in messages))
+            tooltip = ''.join(
+                '<p style="background-color: {}; margin: 0;">'
+                '<nobr style="font-size: {}px;">&nbsp;{}&nbsp;</nobr></p>'.
```

Comment:
Font sizing doesn't work or is too small. Some padding would be nice.

![screenshot - 07292016 - 12 32 49 pm](https://cloud.githubusercontent.com/assets/684364/17245831/b36b5052-5588-11e6-9552-17c5f2182f10.png)